<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect Many</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>821</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pb_iniciar_jogo">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>520</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Iniciar Jogo</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>15</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 6px; 
 } </string>
      </property>
      <property name="title">
       <string>Jogador 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nome_jogador_1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>191</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Jogador 1</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
      <widget class="QWidget" name="widget_cor_jogador_1" native="true">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_cor_jogador_1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>67</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Alterar</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_bg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>501</width>
        <height>541</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagem</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>305</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 6px; 
 } </string>
      </property>
      <property name="title">
       <string>Configurações</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>63</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>linhas</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_linhas">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>43</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>7</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>121</width>
         <height>15</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tamanho do Tabuleiro</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_colunas">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>43</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>19</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>63</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>colunas</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>230</width>
         <height>15</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total de peças alinhadas pra ganhar o jogo</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_pecas_alinhadas">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>43</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 6px; 
 } </string>
      </property>
      <property name="title">
       <string>Jogador 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nome_jogador_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>191</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Jogador 2</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
      <widget class="QWidget" name="widget_cor_jogador_2" native="true">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:blue;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_cor_jogador_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>67</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Alterar</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>232</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect Many</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
